Update development

Tschakki opened the pull request:
### What was the problem?

development branch needs update with main

### How was it tested?

<!--- Please describe how you tested your changes -->

mitsujutsu commented on packages/cryptography/cryptography.md:
````diff
@@ -45,6 +46,30 @@ $ npm install --save @liskhq/lisk-cryptography
+export NACL_FAST=enable
+```
+
+To switch back to [TweetNaCl.js](https://github.com/dchest/tweetnacl-js) library, which is slower but can also be executed on client-side, set it to `disable`:
````
Maybe indicate that the default setting is `tweetnacl` enabled: `To switch back to the default..`

mitsujutsu commented on packages/cryptography/cryptography.md:
```diff
@@ -45,6 +46,30 @@ $ npm install --save @liskhq/lisk-cryptography
+
+The `lisk-cryptography` package can be used on server-side as well as client-side code.
+
+In case `lisk-cryptography` is used server-side, you may want to speed up the application by using the [sodium-native](https://github.com/sodium-friends/sodium-native) library.
```
Leaving a comment here to update this link once we have our own forked version of both `sodium-native` and `tweetnacl` as per [this issue](https://github.com/LiskHQ/lisk-elements/issues/795)

Tschakki committed:
Release 03-04-2024 (#105)

### Changelog?

- #25 
- #41 
- #102 

### How was it tested?

https://liskhq.github.io/lisk-documentation/

Tschakki committed:
Release 04-04-2024 (#110)

* Add batcher address

* Add entrypoint contract

Tschakki committed:
Release 15-04-24 (#114)

### What was the problem?

Includes

- #28 
- #44 
- #90 
- #103 

### How was it tested?

<!--- Please describe how you tested your changes -->

TalhaMaliktz committed:
Add Lisk Mainnet information

Add Lisk Mainnet information

Add Lisk Mainnet information

Add Lisk Mainnet information

Add Lisk Mainnet information

TalhaMaliktz committed:
Apply suggestions from Mona's review

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

TalhaMaliktz committed:
Update docs/building-on-lisk/interacting-with-the-blockchain/viem.mdx

TalhaMaliktz committed:
Update connecting-to-a-wallet.mdx

TalhaMaliktz committed:
Merge branch 'main' into 89-add-network-info-for-lisk-mainnet

TalhaMaliktz committed:
Comment out Governance related information

TalhaMaliktz committed:
Apply suggestions from Himanshu's review

Co-authored-by: Himanshu Nagda <69150921+nagdahimanshu@users.noreply.github.com>

TalhaMaliktz committed:
Apply Himanshu's feedback

TalhaMaliktz committed:
Add Lisk Mainnet information (#122)

### What was the problem?

This PR resolves #89
